### ISSUE TITLE
Bug: white cursor does not play nice with light mode

### ISSUE THREAD

**Issue by chrisgrieser** [clevcode/obsidian-terminal-plugin#8]
basically, in light mode, the cursor is not visible anymore due to being a white cursor on a white background

**Comment by clevcode** [comment]
Yeah, I'm aware of that, it only plays well with "dark mode" themes at the moment :)

The plan is to both make the colorscheme completely configurable, as well as have sane defaults for both dark and light mode themes
